kits/javascript/src/error.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright 2023 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// List of runtime errors
+pub enum RuntimeError {
+    InvalidBinding(String),
+}
```

Comment:
```suggestion
    InvalidBinding(String),
```

Looking at this type it's not very clear to me what is the inner String. We can be more explicit and descriptive at the type-level by doing this:

```suggestion
pub enum RuntimeError {
    InvalidBinding { invalid_export: String },
}
```

Reply:
This also allows us to grow the type information in a backwards and descriptive compatible way.

Reply:
Sounds good! I'm gonna apply it 😄